eclipse-collections/src/main/java/org/eclipse/collections/impl/map/ordered/mutable/UnmodifiableMutableOrderedMap.java

```diff
@@ -1394,6 +1394,12 @@ public void appendString(Appendable appendable, String start, String separator,
         return this.delegate.newEmpty();
     }
 
+    @Override
+    public MutableOrderedMap<K, V> newEmpty(int initialCapacity)
+    {
+        return this.delegate.newEmpty(initialCapacity);
```

Comment:
I am stumped. How does this compile? The generic arguments are missing.

Reply:
I checked and there's a warning on this line about unchecked types, as well as a warning on the parent about the shadowing.

Reply:
My biggest concern is that the build passed. This should have failed TeamCity for sure, and probably a regular javac build with warnings treated as errors.